src/components/InventoryTable/EntityTable.js

```diff
@@ -71,7 +70,7 @@ const EntityTable = ({
   );
 
   const defaultRowClick = (_event, key) => {
-    history.push(
+    navigate(
```

Comment:
I think here we can freely use `navigate` from react-router and just replace with `navigate(key)` (using relative path).